src/components/HydrantList.js

```diff
@@ -27,49 +27,44 @@ class HydrantList extends React.Component {
 
   state = {
     open: true,
-    showDetails:{}
+    showDetails: {}
   }
 
-render() {
-
-  const {
-    classes,
-    editableTrail,
-    hydrants,
-    hydrantDeleted,
-    modifyHydrant
-  } = this.props;
-
-  const trailHydrants = hydrants
-      .filter((h) => h.get('trail') === editableTrail.get('id'))
-      .sort((a,b) => a.get('name').localeCompare(b.get('name'), undefined, {numeric: true}))
-      .valueSeq()
-
-  const toggleHighLight = (feature, state) => {
+  toggleHighLight = (feature, state) => {
     feature.set('highlighted', state)
```

Comment:
Not concerning the code here, but one detail we should clean up as well: in exports, features keep all their properties. I think we should make sure all features have `unset` their highlighted and selected properties since those are just related to application state. 

This is also another reason it's good that we explicitly only set the properties we want on features in those `update__Features` methods in the reducers.

Reply:
Good call, I'll fix this once we merge everything in.